adeolaidowu/MovieApi
# Ft registerlogin

**What does this PR do?**
This PR filters by date range provided in the URL
 
- [x] List of what get done in this PR
     [x] Implementation of the register and login endpoint.

**How can this be tested?**
       - clone this repository
       - Run from visual studio

Expected result
[x] Returns an Ok request if user registration is successful
[x] Returns a Bad request if it is unsuccessful 
[x] Returns an Ok request if user login is successful
[x] Returns a Bad request if it is unsuccessful
